### ISSUE TITLE
What does the format string x-y(n) and variants mean?

### ISSUE THREAD

**Issue by bakoontz** [rohanisaac/spc#8]
The docs are a bit unclear on this...can you explain what these strings mean?

**Comment by rohanisaac** [comment]
Yeah, I've been meaning to get a better explanation for them, It mainly refers to where the data is in the object, which comes from the various ways software can store data in a spc file format. Before the `-` means the data is global, after the `-` means the data is in a subFile, and the (n) refers to the number of subfiles. 

Examples: 
x-y(5) means global x data, 5 corresponding y data series.
-xy(4) means 4 subfiles with individual x and y data series.
gx-y(10) means global x generated data, and 10 corresponding y data series.

The first table under Object Format in the readme is gives the actual elements that exist in each case. I'll try to make that section clearer, it is a little confusing right now.
